PowerEditor/src/Parameters.cpp

```diff
@@ -550,6 +550,9 @@ int getKwClassFromName(const TCHAR *str)
 	if (!lstrcmp(TEXT("type3"), str)) return LANG_INDEX_TYPE3;
 	if (!lstrcmp(TEXT("type4"), str)) return LANG_INDEX_TYPE4;
 	if (!lstrcmp(TEXT("type5"), str)) return LANG_INDEX_TYPE5;
+	if (!lstrcmp(TEXT("type6"), str)) return LANG_INDEX_TYPE6;
+	if (!lstrcmp(TEXT("type7"), str)) return LANG_INDEX_TYPE7;
+	if (!lstrcmp(TEXT("type8"), str)) return LANG_INDEX_TYPE8;
 
 	if ((str[1] == '\0') && (str[0] >= '0') && (str[0] <= '8')) // up to KEYWORDSET_MAX

```

Comment:
Perhaps, `(str[0] <= '9')`?

Reply:
@xylographe LIST_9 is removed in the recent amend, so should be ok.